File: URI-py/1180.py
# -*- coding: utf-8 -*-
n = int(input())
lista = list(map(int,input().split()))
pos = cont = 0
menor = lista[0]
for i in lista:
    if i<menor:
        menor = i
        pos = cont
    cont += 1
print("Menor valor: %d"%menor)
print("Posicao: %d"%pos)

File: URI-cpp/1180.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
    vector<int>lista;
    int n,m;
    cin >> n;
    for (int i = 0; i < n; ++i) {
        cin >> m;
        lista.push_back(m);
    }
    auto min = min_element(lista.begin(),lista.end());
    auto minindex = min_element(lista.begin(),lista.end()) - lista.begin();;
    printf("Menor valor: %d\n",*min);
    printf("Posicao: %d",minindex);

    return 0;
}
